apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -65,63 +66,68 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
 
             pluginFinder = new PluginFinder(new PluginBootstrap().loadPlugins());
         } catch (AgentPackageNotFoundException ape) {
-            logger.error(ape, "Locate agent.jar failure. Shutting down.");
+            LOGGER = LogManager.getLogger(SkyWalkingAgent.class);
```

Comment:
You should make `SnifferConfigInitializer.initializeCoreConfig(agentArgs);` in a separate `try/catch/finally` to reset the logger, please do this so many times.

Reply:
Pls check again the current modification